translations/handleExternalBranchedAndroidTranslations.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+
+# verbose and exit on error
+set -xe
+
+# import GPG keys
+gpg --import /gpg/nextcloud-bot.public.asc
+gpg --allow-secret-key-import --import /gpg/nextcloud-bot.asc
+gpg --list-keys
+
+# fetch git repo
+git clone git@github.com:$1/$2 /app
+
+# switch to translation branch
+git checkout $3
```

Comment:
If the branch doesn't exist this will cause issues. So maybe we need to create the branch locally only and then do the force push at the end to `git push -f origin $3`